examples/btc-avg/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "btc-avg"
```

Comment:
we don't want the btc-avg code here.. that is a different task for a different crate. This one is just for the square example (so the whole https stuff can be punted for now)

I assume you are closing https://github.com/Lay3rLabs/avs-toolkit/issues/16 with this PR - good to mention that in the description for clarity and to auto-close the issue